src/modules/auth/auth.service.test.ts

```diff
@@ -0,0 +1,42 @@
+import { ConfigService } from '@nestjs/config';
+import { Test, TestingModule } from '@nestjs/testing';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+
+import { AuthService } from './auth.service';
+
+describe('AuthService', () => {
+  let authService: AuthService;
```

Comment:
Do we need to add test cases where `api-strategy` is also null and void?

Reply:
the strategy piece got updated now and it will always come from a Constant.